docs/deploying-cf-locally.md

```diff
@@ -0,0 +1,348 @@
+# How to deploy Cloud Foundry in Bosh-Lite using cf-deployment
+Deploying Cloud Foundry with Bosh-Lite is a low-cost, lightweight approach tailored for development and testing environments. Unlike AWS deployments, Bosh-Lite runs the entire Cloud Foundry stack inside a single VM on your local machine using BOSH’s local CPI (warden). This drastically reduces infrastructure requirements and speeds up prototyping. Use this setup when you need an isolated, disposable CF environment for debugging, experimentation, or learning.
+
+> 💡 Note: This environment is not production-grade.
+
+## Install VirtualBox and prerequisites
+ 
+Ensure that VirtualBox and its extension pack are installed and configured
+properly.
+
+```bash
+sudo dnf install virtualbox
+VBoxManage extpack install --replace Oracle_VirtualBox_Extension_Pack-7.1.8.vbox-extpack
+VBoxManage list extpacks
```

Comment:
what's the expected result of running this command? is it necessary to complete the installation or is it just to verify that the packs were installed? if so, what do you think of adding the output as an example?